plugin/restore/mysql/mysql.go

```diff
@@ -680,6 +680,221 @@ func (r *Restore) GetSortedBinlogFilesMetaOnServer(ctx context.Context) ([]Binlo
 	return sortBinlogFiles(binlogFiles), nil
 }
 
+// ConvertTsToLocalBinlogCoordinate converts a timestamp to binlog coordinate using local binlog files.
+// We do all kinds of comparison like finding proper backup using the converted BinlogInfo, but not the original timestamp.
```

Comment:
The second sentence doesn't sound. We can remove it.

Reply:
SG.